### PR TITLE
💹 Fix chunk sizes

### PR DESCRIPTION
**Pull request by WRadoslaw**
#4575

### CODE REVIEW

File: packages/atlas/src/MainLayout.tsx [hunk 1]
```diff
@@ -108,12 +108,14 @@ export const MainLayout: FC = () => {
   return (
     <>
       <CookiePopover />
+      {/*<Suspense fallback={<div>adawd</div>}>*/}
```
Comment (attemka) [inline packages/atlas/src/MainLayout.tsx:111]: suspense will be included when it'll be supported by apollo client

File: packages/atlas/src/components/StudioEntrypoint/StudioEntrypoint.tsx [hunk 2]
```diff
@@ -35,3 +35,5 @@ export const StudioEntrypoint: FC<StudioEntrypointProps> = ({ enterLocation }) =
 
   return <StudioLoading />
 }
+
+export default StudioEntrypoint
```
Comment (attemka) [inline packages/atlas/src/components/StudioEntrypoint/StudioEntrypoint.tsx:39]: we should't change it in this PR, but generally we should avoid default export in files

**Review by attemka — APPROVED** [review]

File: packages/atlas/src/components/StudioEntrypoint/StudioEntrypoint.tsx (hunk 2 above)
Comment (WRadoslaw) [reply packages/atlas/src/components/StudioEntrypoint/StudioEntrypoint.tsx:39]: `React.lazy` requires default export

File: packages/atlas/src/MainLayout.tsx (hunk 1 above)
Comment (WRadoslaw) [reply packages/atlas/src/MainLayout.tsx:111]: This should be deleted, since I introduced it in `Viewer` and `Studio` layout

**Pull request merged**